Dear/Dear/Controller/LetterViewController.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  LetterViewController.swift
+//  Dear
+//
+//  Created by 한연희 on 2022/07/17.
+//
+
+import UIKit
+
+class LetterViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.hideKeyboardWhenTappedAround()
+        // Do any additional setup after loading the view.
+    }
+
+    @IBAction func PrintAlert(_ sender: UIButton) {
+        let alert = UIAlertController(title: "마음카드를 보낼까요?", message: "발송 완료 후 수정할 수 없습니다", preferredStyle: .alert)
+        let yes = UIAlertAction(title: "확인", style: .default, handler: nil)
+        let no = UIAlertAction(title: "취소", style: .destructive, handler: nil)
+        
+        alert.addAction(no)
+        alert.addAction(yes)
```

Comment:
```suggestion
        [no, yes].forEach { alert.addAction($0) }
```

forEach를 활용하면 반복된 작업을 한줄로 읽기편하게 수정 가능하답니다 ~

Reply:
넵넵! 감사합니다